src/bot.coffee

```diff
@@ -171,30 +171,38 @@ class SlackBot extends Adapter
   # @private
   ###
   eventHandler: (event) =>
-    {user, channel, bot} = event
+    {user, channel} = event
 
     # Ignore anything we sent
     # NOTE: coupled to getting `rtm.start` data
-    return if (user && (user.id is @self.id || user.id is @self.bot_id)) || (bot && (bot.id is @self.bot_id))
+    return if user && (user?.id is @self.id)
+
+    
+    ###*
+    # Hubot user object in Brain.
+    # User can represent a Slack user OR bot
```

Comment:
nitpicky, but i think you should say "Slack human user or bot user", and also make it clear that the value is an instance of Hubot's `User` class